### DISCUSSION TITLE
How to enable apm customize enhance plugin when using a Java agent injector?

### DISCUSSION THREAD

**Discussion by saowu** [apache/skywalking#11952] [Q&A]
skywalking-swck version: v0.8.0

I found that the official website documentation does not provide comments for configuration, and it is not possible to modify agent.config through configmap to add plugin.customize.enhance_file because this method has been deprecated!

https://skywalking.apache.org/docs/skywalking-swck/v0.8.0/java-agent-injector/
https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/configurations/#table-of-agent-configuration-properties

![image](https://github.com/apache/skywalking/assets/33569619/b3243e57-ac62-437b-a963-48b29e0a2e3d)

**Comment by tanshanshan** [comment]
you mean how to modify agent.config in skywalking-swck
the first url you have paste [the-ways-to-configure-the-agent](https://skywalking.apache.org/docs/skywalking-swck/v0.8.0/java-agent-injector/#the-ways-to-configure-the-agent)

> The java agent injector supports a precedence order to configure the agent:
Annotations > SwAgent > Configmap (Deprecated) > Default Configmap (Deprecated)
